eng/pipelines/templates/jobs/ci.yml

```diff
@@ -67,16 +67,25 @@ jobs:
           ServiceDirectory: ${{ parameters.ServiceDirectory }}
           TestPipeline: ${{ parameters.TestPipeline }}
 
-  - ${{ if ne(parameters.RunUnitTests, false) }}:
-    - template: /eng/common/pipelines/templates/jobs/archetype-sdk-tests-generate.yml
-      parameters:
-        JobTemplatePath: /eng/pipelines/templates/jobs/ci.tests.yml
-        MatrixConfigs: ${{ parameters.MatrixConfigs }}
-        MatrixFilters: ${{ parameters.MatrixFilters }}
-        MatrixReplace: ${{ parameters.MatrixReplace }}
-        CloudConfig:
-          Cloud: Public
-        AdditionalParameters:
+  - job: Compliance
```

Comment:
Don't know enough to approve but just want to say I am very excited to have this!